pkgs/os-specific/darwin/apple-sdk-12.3/frameworks/fixups.nix

```diff
@@ -0,0 +1,187 @@
+# TODO(@connorbaker): Copied from 11.0 -- should be regenerated.
```

Comment:
```suggestion
```
?? should it be regenerated? regardless line should be removed.

Reply:
I’ll drop the to-dos. I don’t plan on regenerating these files if they work.